sql/synth_recipes.sql

```diff
@@ -2083,7 +2083,7 @@ INSERT INTO `synth_recipes` VALUES (29951,1,0,0,0,55,0,0,0,0,0,4100,4242,13478,0
 -- INSERT INTO `synth_recipes` VALUES (29952,1,0,0,0,21,0,0,0,0,0,4100,4242,17399,0,0,0,0,0,0,0,648,648,818,818,1,2,2,2,'Sabiki Rig (desynth)',NULL); -- 2022.03.04 Base on FFXIclopedia removed Hq2 stated copper ingot X 3 strange result and flag needing verification
 INSERT INTO `synth_recipes` VALUES (29953,1,0,0,0,66,0,0,0,0,0,4100,4242,12673,0,0,0,0,0,0,0,817,850,653,1228,6,4,1,6,'Mythril Gauntlets (desynth)',NULL); -- 2022.03.04 Base on FFXIclopedia
 INSERT INTO `synth_recipes` VALUES (29954,1,0,0,0,48,0,0,0,0,0,4100,4242,12426,0,0,0,0,0,0,0,648,850,850,850,1,1,1,1,'Banded Helm (desynth)',NULL); -- BGWiki desynth yield incomplete, but does not contradict this
--- INSERT INTO `synth_recipes` VALUES (29955,1,0,0,0,85,0,0,0,0,0,4100,4242,13352,0,0,0,0,0,0,0,746,786,786,786,1,1,1,1,'Ruby Earring (desynth)',NULL); -- 2022.03.04 Base on FFXIclopedia Level Unknown Hq3 rubyX2 strange result
+INSERT INTO `synth_recipes` VALUES (29955,1,0,0,0,85,0,0,0,0,0,4100,4242,13352,0,0,0,0,0,0,0,786,786,746,746,1,1,1,2,'Ruby Earring (desynth)',NULL); -- Based off other 85 Goldsmith earring desynths
```

Comment:
This would make it so a desynth yields, by HQ tier order:
- 1 ruby
- 1 ruby
- 1 platinum ingot
- 2 platinum ingot

I see that you got this information from the sapphire earring, which both BG wiki and clopedia conflict in results.
In good faith, I have to ask you keep this commented out until we can confirm it ourselves in retail